rustup-init.sh

```diff
@@ -33,7 +33,7 @@ RUSTUP_QUIET=no
 # NOTICE: If you change anything here, please make the same changes in setup_mode.rs
 usage() {
     cat <<EOF
-rustup-init 1.27.1 (a8e4f5c64 2024-04-24)
+rustup-init 1.28.0 (b6fdf2fa 2024-11-25)
```

Comment:
Would it not be better that the release commit include the full changelog? Even though it has no effect on the binary itself.

Reply:
@chrisdenton Ideally we'd be able to do that, but not just yet. The plan is that this will be a placeholder for the release team to fill in. For now we just want the traceability.

Reply:
Also given that this is just the beta release, I don't think it is as important for right now.